src/components/common/ColonyDecisions/NewDecisionButton/NewDecisionButton.tsx

```diff
@@ -1,28 +1,30 @@
 import React from 'react';
 
 import { useDialog } from '~shared/Dialog';
-// import DecisionDialog from '~dashboard/Dialogs/DecisionDialog';
+import { DecisionDialog } from '~common/ColonyDecisions';
 // import RemoveDraftCreateNewDecision from '~dashboard/Dialogs/RemoveDraftDecisionDialog';
 
 import { DialogButton } from '~shared/Button';
-import { useAppContext, useColonyContext } from '~hooks';
+import { useAppContext } from '~hooks';
+import useRichTextEditor from '~hooks/useRichTextEditor';
 
-const displayName = 'common.NewDecisionButton';
+const displayName = 'common.ColonyDecisions.NewDecisionButton';
 
 interface Props {
   ethDomainId: number;
 }
 
 const NewDecisionButton = ({ ethDomainId }: Props) => {
-  const { colony } = useColonyContext();
   const { user } = useAppContext();
+  const { editor } = useRichTextEditor();
```

Comment:
Instantiating the editor here and passing it down ensures it's defined and available before the dialog opens. This prevents the dialog from opening before the editor object is available, which results in a jiggery ui.